src/s3path.jl

```diff
@@ -123,7 +130,8 @@ function Base.:(==)(a::S3Path, b::S3Path)
 end
 
 function Base.getproperty(fp::S3Path, attr::Symbol)
-    if attr === :anchor
+    if attr 
+      :anchor
```

Comment:
Looks to be a mistake:
```suggestion
    if attr === :anchor
```